src/renderer/components/Config/PrereqsView.tsx

```diff
@@ -91,6 +91,44 @@ const PrereqsView = ({ sx }: Props) => {
           .
         </Typography>
       )
+    } else if (status.id === 'ps1ExecutionPolicy') {
+      status.description = (
+        <Typography fontSize={14}>
+          <span style={{ fontSize: 14, opacity: 0.6 }}>
+            {
+              // Check if the user agent contains PowerShell version information
+              window.navigator.userAgent.includes('PowerShell')
+                ? `Run the following command in PowerShell ${window.navigator.userAgent}:`
+                : 'Run the following command in PowerShell:'
+            }
+          </span>
+          <br />
+          <br />
+          <span style={{ fontSize: 14, opacity: 0.6 }}>
+            Check whether the execution policy is set to allow unsigned PowerShell scripts.
+          </span>
+          <br />
+          <br />
+          <span style={{ fontSize: 14, opacity: 0.6 }}>
+            Afterwards, if the execution policy is not set to allow unsigned PowerShell scripts, you can do so by
+            running the following commands:
+            <br />
+            <br />
+            <code>Get-ExecutionPolicy</code> - This command will show you the current execution policy.
+            <br />
+            <br />
+            <code>Set-ExecutionPolicy Unrestricted</code> - Use this command to set the execution policy to allow
```

Comment:
Just keep the script. Remove - Use this command to set the execution policy to allow